TicTacToeForBot.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[52]:
+
+
+import random
+
+def drawBoard(board):
+    #Эта функция рисует игровую доску с выполненными ходами
+
+     #"Доска" является списком из 10 строк которые рисуют доску в 
+#символьной графике
+    print(' | |')
```

Comment:
It's better to use a multiline python string

Reply:
Done